python/lsst/ctrl/execute/slurmPlugin.py

```diff
@@ -159,8 +163,10 @@ def submit(self, platform, platformPkgDir):
             print("The user home directory is %s " % self.getUserHome())
 
         if auto:
-            self.glideinsFromJobPressure(generatedSlurmFile)
+            self.glideinsFromJobPressure(platformPkgDir)
         else:
+            generatedSlurmFile = self.createFilesFromTemplates(platformPkgDir)
+            cmd = "sbatch --mem %s %s" % (totalMemory, generatedSlurmFile)
```

Comment:
If I didn't miss something, I'd move the cmd assignment down to right before that glide for loop where it is used.

Reply:
Well I'll have to think about that; I have it together with Slurm submit file retrieval, so both may have to move at some point.